src/axom/primal/tests/primal_octahedron.cpp

```diff
@@ -23,9 +23,9 @@ class OctahedronTest : public ::testing::Test
 public:
   static const int DIM = 3;
 
-  typedef double CoordType;
-  typedef primal::Point<CoordType, DIM> QPoint;
-  typedef primal::Octahedron<CoordType, DIM> QOct;
+  using CoordType = double;
```

Comment:
is it possible to have a chang-tidy rule to catch this sort of thing?

Reply:
These `typedef` -> `using` changes were from the `modernize-use-using` rule

Reply:
That was actually the impetus for revisiting clang-tidy in this PR -- I had been manually updating the `typedef`s when I updated old `primal` test files (e.g. in #1251) and thought clang-tidy would have a `modernize-*` rule to automate it.